solutions/standard/variables.tf

```diff
@@ -159,7 +172,7 @@ variable "configuration" {
     tcp_keepalives_interval    = 15
     tcp_keepalives_count       = 6
     archive_timeout            = 1800
-    wal_level                  = "replica"
+    wal_level                  = "logical"
```

Comment:
no don't do this - there is a backend bug. We need to remove `wal_level` all together from the configuration so it defaults to `replica`. I think @shemau is going to make the change in https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/pull/513

Reply:
actually its coming in https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/pull/514